### PR TITLE
lldpd: Ensure "/usr/local"/var/run exists

### PR DESCRIPTION
**Pull request by spkane**
Ensure that /var/run exists under the homebrew root. If not, lldpd will not start.

### CODE REVIEW

File: Library/Formula/lldpd.rb [hunk 1]
```diff
@@ -20,6 +20,8 @@ class Lldpd < Formula
   depends_on "jansson"  if build.with? "json"
 
   def install
+    (var/"run").mkpath
```
Comment (mistydemeo) [inline Library/Formula/lldpd.rb]: Instead of putting this in `install`, can you do it in `postinstall`? That way it should also work for binary packages.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `lldpd: ensure var/run exists.`.

File: Library/Formula/lldpd.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/lldpd.rb]: Maybe we should add an audit check to unify this. We've mixed and matched this between postinstall and install all over the place.

Comment (mikemcquaid) [inline Library/Formula/lldpd.rb]: @DomT4 :+1:

**Pull request closed without merging**